src/ui/screens/settings/ViewReset.qml

```diff
@@ -0,0 +1,189 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import Mozilla.VPN 1.0
+import Mozilla.Shared 1.0
+import components 0.1
+import compat 0.1
+import "qrc:/ui/sharedViews"
+
+ViewFullScreen {
+    property string _menuTitle: MZI18n.ResetSettingsResetLabel
+    property var _menuOnBackClicked:  () => {
+                                          getHelpStackView.pop();
+                                          navbar.visible = true
+                                      }
+
+    content: ColumnLayout {
+        spacing : 0
+
+        Image {
+            id: icon
+
+            Layout.alignment: Qt.AlignHCenter
+
+            sourceSize.height: 80
+            sourceSize.width: 80
+            fillMode: Image.PreserveAspectFit
+            source: "qrc:/ui/resources/reset.svg"
+        }
+
+        MZHeadline {
+            Layout.topMargin: 24
+            Layout.fillWidth: true
+
+            text: MZI18n.ResetSettingsTitle
+        }
+
+        MZInterLabel {
+            Layout.topMargin: 8
+            Layout.fillWidth: true
+
+            text: MZI18n.ResetSettingsBody1
+            color: MZTheme.theme.fontColor
+            horizontalAlignment: Text.AlignLeft
+        }
+
+        MZInterLabel {
+            Layout.topMargin: 16
+            Layout.fillWidth: true
+
+            text: "<ul style='margin-left: -20px;'><li>%1</li></ul>".arg(MZI18n.HelpSheetsPrivacyTitle)
```

Comment:
Why are we styling text inline?

Reply:
It's markdown to get a bulleted list

Reply:
Do we do similar things other places in the app? I don't see many. Is there any cleaner way to do a bulleted list? (Genuine question, I'm not familiar w/ Qt best practices.)

Reply:
I believe it's a standard practice. Alternatively we could build something ourselves like a rounded rectangle + text side by side, but I'm not sure that is any more beneficial. We use the same technique in a few places like guides and in-app messages, but, afaik we don't have bulleted list anywhere else

Reply:
Curious: Did you happen to try this with a screen reader? I'm wondering how Qt exposes the text value and if the markup could cause any grief there.